config.default.ini.php

```diff
@@ -11,6 +11,11 @@
 ; false = disabled (default)
 custom_timeout = false
 
+; Ignore specified custom timeout for specific requests.
+; true  = enabled
+; false = disabled (default)
+ignore_custom_timeout = false
```

Comment:
This parameter is unnecessary. If `custom_timeout = false`, we can just imply `ignore_custom_timeout = true`.